From 203785526af7308a799401c0b037db409269119c Mon Sep 17 00:00:00 2001
From: Adam Williamson <awilliam@redhat.com>
Date: Thu, 10 May 2018 11:00:17 -0700
Subject: [PATCH 1/2] Fix a missed variable rename in ScanDirectory (caused a
 crash)

halfline ran into gtk-doc crashing when he was trying to cut an
accountsservice release; looking into it we found that the first
arg to ScanDirectory was renamed from `dir` to `scan_dir` in
9292e0a (to avoid overriding a builtin, I guess) but this one
reference to it was not changed. This should fix it.

Signed-off-by: Adam Williamson <awilliam@redhat.com>
---
 gtkdoc/rebase.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gtkdoc/rebase.py b/gtkdoc/rebase.py
index 424c3e6..2a1d495 100755
--- a/gtkdoc/rebase.py
+++ b/gtkdoc/rebase.py
@@ -108,7 +108,7 @@ def ScanDirectory(scan_dir, options):
 
         if onlinedir and entry == "index.sgml":
             log(options, "Reading index from index.sgml")
-            onlinedir = ReadIndex(dir, entry)
+            onlinedir = ReadIndex(scan_dir, entry)
             have_index = True
         elif entry == "index.sgml.gz" and not os.path.exists(os.path.join(scan_dir, 'index.sgml')):
             # debian/ubuntu started to compress this as index.sgml.gz :/
-- 
2.17.0

